packages/harmony/src/components/hover-card/HoverCardHeader.tsx

```diff
@@ -11,7 +8,8 @@ import { HoverCardHeaderProps } from './types'
 export const HoverCardHeader = ({
   icon,
   title,
-  onClose
+  onClose,
+  iconRight
```

Comment:
if it makes sense to default iconRight = IconArrowRight, maybe do that?

Reply:
the cosign / remix contest flairs don't have an icon here so that's why i modified this file